<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Pictures">
       <attribute name="title">
        <string>Pictures</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Input Directories</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="pictureInputPaths"/>
            </item>
            <item>
             <widget class="QPushButton" name="pictureInputButton">
              <property name="text">
               <string>Open Input Directory</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>File Extensions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="picturExtList"/>
            </item>
            <item>
             <widget class="QCheckBox" name="isPictureAllFiles">
              <property name="text">
               <string>All file types</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pictureFileExtEdit"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="addPictureFileExtButton">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="removePictureFileExt">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pictures_sorting_options">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="isPictureDateSorted">
            <property name="text">
             <string>Sort by creation date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="isPictureFileComments">
            <property name="text">
             <string>Preserve original file path in file comments</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="output_horizontal">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="pictureDestinationField">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pictureDestination">
            <property name="text">
             <string>Pictures Destination</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="unifiedOutputField">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outputDestination">
        <property name="text">
         <string>Output Destination</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearUnifiedOutput">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="process">
        <property name="toolTipDuration">
         <number>2</number>
        </property>
        <property name="whatsThis">
         <string>Begin file copy and sort</string>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
